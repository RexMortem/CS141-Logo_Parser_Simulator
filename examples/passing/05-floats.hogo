; Distances and rotations are not always whole numbers.

forward 30.5
right 140.2
back 200.6