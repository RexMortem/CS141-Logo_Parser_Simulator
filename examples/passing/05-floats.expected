[GoForward 30.5, TurnRight 140.2, GoBackward 200.6]